Godot Project/Scenes/Levels/Level One/level_one.tscn

```diff
@@ -321,6 +325,9 @@ rotation = 3.14159
 [node name="Player" parent="." index="10"]
 position = Vector2(-226, 255)
 
+[node name="Camera2D" parent="Player" index="6"]
+position_smoothing_enabled = false
```

Comment:
Looks like level one does not inherit the template so this override has to stay. I will refactor the level to inherit template if time allows.